src/components/Duplicate.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Duplicate = () => {
+  return (
+    <div>
+      <p style={{ color: 'red' }}>
+        ❌ <b>That item is already in your list</b>
+      </p>
+    </div>
+  );
+};
+
```

Comment:
I really like the way you approached this AC. If there were additional places in the code that required error messages, I think a small refactor would make this reusable for multiple situations.